src/client.rs

```diff
@@ -390,6 +409,49 @@ impl HttpClientBuilder {
         self
     }
 
+    /// Set a custom SSL/TLS CA certificate bundle to use for all client connections.
+    ///
+    /// The default value is none.
+    ///
+    /// Note: for Windows, setting `no_revoke(true)` might also be nessery.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use isahc::config::*;
+    /// # use isahc::prelude::*;
+    /// #
+    /// let client = HttpClient::builder()
+    ///     .ca_certificate("ca.pem".into())
+    ///     .build()?;
+    /// # Ok::<(), isahc::Error>(())
+    /// ```
+    pub fn ca_certificate(mut self, ca_path: PathBuf) -> Self {
```

Comment:
To support the potential of using certificates from sources other than a file path without breaking changes, I would just expose `CaCertificate` here directly. Example:

```rust
let client = HttpClient::builder()
    .ssl_ca_certificate(CaCertificate::path("ca.pem"))
    .build()?;
```

This would allow us to add new constructors to `CaCertificate` in the future without breaking changes.